Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string firstName = PromptString("Enter your first name: ");
+            string lastName = PromptString("Enter your last name: ");
+            int age = PromptInt("Enter your age: ");
+            float weight = PromptFloat("Enter your weight (in kg): ");
+            float height = PromptFloat("Enter your height (in meters): ");
+            float bmi = CalculateBmi(weight, height);
+
+            Console.WriteLine("{0} {1} is {2} years old, he has a BMI of {3} with a weight of {4} kg and a height of {5} meters.", firstName, lastName, age, bmi, weight, height);
+         }
+
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            // TODO: check input can be parsed as int
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)